test/test_base_service.py

```diff
@@ -648,10 +648,10 @@ def test_files_dict():
     service = AnyServiceV1('2018-11-20', authenticator=NoAuthAuthenticator())
 
     form_data = {}
-    file = open(
+    with open(
```

Comment:
The linter updated and complained about not using `with` here. This resolves that.